app/controllers/games_controller.rb

```diff
@@ -1,2 +1,26 @@
 class GamesController < ApplicationController
+
+	def index
+		@games = Game.all
+		@open_games = Game.where(:black_uid => nil)
```

Comment:
Yes, in the task it said we were assigning a user joining a game to
black_uid, so my assumption was the game creator would have been assigned
white_uid.

--
Jeff Powers
Global Partners, Inc.
www.globalpartnersinc.com

Connect on Linkedin: http://www.linkedin.com/in/jeffreytpowers/

On Thu, Sep 10, 2015 at 9:26 AM, Jeff Durand <notifications@github.com>
wrote:

> In app/controllers/games_controller.rb
> <https://github.com/Firehose-Team-Awesome/chess-game/pull/14#discussion_r39158935>
> :
>
> > @@ -1,2 +1,26 @@
> >  class GamesController < ApplicationController
> > +
> > +	def index
> > +		@games = Game.all
> > +		@open_games = Game.where(:black_uid => nil)
>
> This won't capture games that don't have a white_uid. My guess is that you
> are going to always assign the first user to the white side of the board.
> This works but removes the option for users to choose a color. Just want to
> make sure you understand the tradeoff.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Firehose-Team-Awesome/chess-game/pull/14/files#r39158935>
> .
>